catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/synchronizations/CompletionSynchronization.java

```diff
@@ -49,6 +51,11 @@ public final void onFailure(Exchange exchange) {
     if (!connected) {
       LOGGER.debug(
           "a network error occurred, The file [{}] failed to process", asyncFileEntry.getName());
+    } else if (exchange.getException() != null) {
+      LOGGER.debug(
```

Comment:
❓ Should this be higher than debug? I'm not sure how frequently this would occur or if it would be considered noise.